src/main/java/com/rabbitmq/client/impl/recovery/AutorecoveringConnection.java

```diff
@@ -782,11 +790,7 @@ public void recoverQueue(final String oldName, RecordedQueue q, boolean retry) {
                         this.propagateQueueNameChangeToConsumers(oldName, newName);
                         // bug26552:
                         // remove old name after we've updated the bindings and consumers,
-                        // plus only for server-named queues, both to make sure we don't lose
-                        // anything to recover. MK.
-                        if(q.isServerNamed()) {
-                            deleteRecordedQueue(oldName);
-                        }
+                        deleteRecordedQueue(oldName);
```

Comment:
wanted to call out that this is removing the q.isServerNamed() check here. But i don't think that is an issue as i don't believe their would have historically been anyway for the name to change unless it was server named.